### PR TITLE
Klein21 patch 4

### PR DESCRIPTION
**Pull request by klein21**
Updating Publications to 2021 on PCMDI website

@mauzey1 
@durack1

### CODE REVIEW

File: publications.md [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+*   Griffies, S. M., G. Danabasoglu, **P. J. Durack**, A. J. Adcroft, V. Balaji, C. Boening, E. P. Chassignet, E. Curchitser, J. Deshayes, H. Drange, B. Fox-Kemper, **P. J. Gleckler**, J. M. Gregory, H. Haak, R. W. Hallberg, H. T. Hewitt, D. M. Holland, T. Ilyina, J. H. Junglaus, Y. Komuro, J. P. Krasting, W. G. Large, S. J. Marsland, S. Masina, T. J. McDougall, J. C. Orr, A. Pirani, F. Qiao, R. J. Stouffer, **K. E. Taylor**, H. Tsujino, P. Uotila, M. Valdivieso, M. Winton and S. G. Yeager, (2016): Experimental and diagnostic protocol for the physical component of the CMIP6 Ocean Model Intercomparison Project (OMIP). _Geoscientific Model Development_, **9**, 3231-3296. doi: [10.5194/gmd-9-3231-2016](http://doi.org/10.5194/gmd-9-3231-2016)
+*   Meehl, G.A., A. Hu, **B.D. Santer**, and S.-P. Xie, 2016: Interdecadal Pacific Oscillation contributions to multi-decadal variability of 20th century globally averaged surface temperatures
```
Comment (mauzey1) [inline publications.md:184]:
```suggestion
*   Meehl, G.A., A. Hu, **B.D. Santer**, and S.-P. Xie, 2016: Interdecadal Pacific Oscillation contributions to multi-decadal variability of 20<sup>th</sup> century globally averaged surface temperatures
```

File: publications.md [hunk 2]
```diff
@@ -0,0 +1,459 @@
...
+*   Fyfe, J.C., G.A. Meehl, M.H. England, M.E. Mann, **B.D. Santer**, G.M. Flato, E. Hawkins, N.P. Gillett, S.-P. Xie, Y. Kosaka, and N.C. Swart, 2016: Making sense of the early 2000s global warming slowdown, _Nature Climate Change_, **6**, 224-228. doi:[10.1038/nclimate2938](http://dx.doi.org/10.1038/nclimate2938)
+*   **Ivanova, D. P., P. J. Gleckler, K. E. Taylor, P. J. Durack, K. D. Marvel**, 2016: Moving beyond the total sea ice extent in gauging model biases, _Journal of Climate_, **29**, 8965-8987. doi:[10.1175/JCLI-D-16-0026.1](http://dx.doi.org/10.1175/JCLI-D-16-0026.1)
+*   Clark, P.U., J.D. Shakun, S.A. Marcott, A.C. Mix, M. Eby, S. Kulp, A. Levermann, G.A. Milne, P.L. Pfister, **B.D. Santer**, D.P. Schrag, S. Solomon, T.F. Stocker, B.H. Strauss, A.J. Weaver, R. Winkelmann, D. Archer, E. Bard, A. Goldner, K. Lambeck, R.T. Pierrehumbert, and G.-K. Plattner, 2016: Consequences of 21st century policy for multi-millennial climate and sea-level change, _Nature Climate Change_, **6**, 360-369
```
Comment (mauzey1) [inline publications.md:187]:
```suggestion
*   Clark, P.U., J.D. Shakun, S.A. Marcott, A.C. Mix, M. Eby, S. Kulp, A. Levermann, G.A. Milne, P.L. Pfister, **B.D. Santer**, D.P. Schrag, S. Solomon, T.F. Stocker, B.H. Strauss, A.J. Weaver, R. Winkelmann, D. Archer, E. Bard, A. Goldner, K. Lambeck, R.T. Pierrehumbert, and G.-K. Plattner, 2016: Consequences of 21<sup>st</sup> century policy for multi-millennial climate and sea-level change, _Nature Climate Change_, **6**, 360-369
```

**Review by mauzey1 — CHANGES_REQUESTED** [review]
There were some details that I missed when generating a Markdown file from the original HTML of publications.html.  The superscript tags didn't carry over for some of the titles.  I have added them below.

File: publications.md (hunk 2 above)
Comment (klein21) [reply publications.md:187]: @mauzey1 

Good to catch these.   Do I need to do anything to make sure they get incorporated?

**Review by mauzey1 — APPROVED** [review]

File: publications.md (hunk 2 above)
Comment (klein21) [reply publications.md]: @mauzey1 

Thanks for the tip about "Commit".  I will do so, if the situation arises again.

**Pull request merged**